flask/serializers.py

```
@@ -0,0 +1,140 @@
+from models import Users, Movies, Cinemas, Screens, Seats, Showtimes, Reservations, Tickets, SeatLocks
+from decimal import Decimal
```

Ruff (0.11.9)

`decimal.Decimal` imported but unused

Remove unused import: `decimal.Decimal`

(F401)